test/features/install_trusty.feature

```diff
@@ -0,0 +1,29 @@
+@trusty
+@infinispan
+@kafka
+Feature: Kogito Trusty
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed with Infinispan and Kafka operators
+
+  @smoke
+  Scenario: Install Kogito Trusty
+    When Install Kogito Trusty with 1 replicas
+    Then Kogito Trusty has 1 pods running within 10 minutes
+    
+#####
+
+  @externalcomponent
+  @infinispan
+  Scenario: Install Kogito Trusty with persistence using external Infinispan
+    Given Infinispan instance "external-infinispan" is deployed with configuration:
+      | username | developer |
+      | password | mypass |
+
+    When Install Kogito Trusty with 1 replicas with configuration:
+      | infinispan | username | developer                 |
+      | infinispan | password | mypass                    |
+      | infinispan | uri      | external-infinispan:11222 |
+
+    Then Kogito Trusty has 1 pods running within 10 minutes
```

Comment:
Please add also a test to check with External Kafka

Reply:
Hi @radtriste , I have an issue with the existing test, I'm trying to figure out what is the problem. 
Since we don't have yet an example in `kogito-example` that we can use here, I'm just checking that the pod is running. Once the example using the `tracing-addon` is added, I'll create another PR to use it here in these tests

Reply:
@r00ta we don't have an example to test this feature yet?

Reply:
Hi @radtriste , I have some trubles with the basic test 'cause it is failing with 
```
{"level":"info","T":"2020-07-31T10:54:09.544Z","logger":"jenkins-cucumber-666b","msg":"Step Install Kogito Trusty with 1 replicas"}

{"level":"info","T":"2020-07-31T10:54:09.544Z","logger":"jenkins-cucumber-666b","msg":"trusty install cli with 1 replicas"}

{"level":"info","T":"2020-07-31T10:54:09.544Z","logger":"jenkins-cucumber-666b","msg":"Execute command /home/jenkins/agent/workspace/Kogito/kogito-operator-pr-check/build/_output/bin/kogito [install trusty --image quay.io/kiegroup/kogito-trusty-nightly:latest --replicas 1 -p jenkins-cucumber-666b]"}

{"level":"error","T":"2020-07-31T10:54:09.690Z","logger":"jenkins-cucumber-666b","msg":"output command: Error while trying to create a new Kogito Trusty Service: the server could not find the requested resource (post kogitotrusties.app.kiegroup.org) \n"}

{"level":"error","T":"2020-07-31T10:54:09.690Z","logger":"jenkins-cucumber-666b","msg":"error output command: infinispan-url not informed, Infinispan will be automatically deployed via Infinispan Operator\nNo Kafka information has been given. A Kafka instance will be automatically deployed via Strimzi Operator in the namespace. Kafka Topics will be created accordingly if they don't exist already\nError: Error while trying to create a new Kogito Trusty Service: the server could not find the requested resource (post kogitotrusties.app.kiegroup.org) \nUsage:\n  kogito install trusty [flags]\n\nExamples:\ntrusty -p my-project\n\nFlags:\n      --channel string                Install Kogito operator from Operator hub using provided channel, e.g. (alpha/dev-preview) (default \"alpha\")\n  -e, --env stringArray               Key/Pair value environment variables that will be set to the service. For example 'MY_VAR=my_value'. Can be set more than once.\n  -h, --help                          help for trusty\n      --http-port int32               Define port on which service will listen internally (default 8080)\n  -i, --image string                  The image which should be used to run Service. For example 'quay.io/kiegroup/kogito-data-index:latest'\n      --infinispan-authrealm string   The Infinispan Server Auth Realm for authentication, example: ApplicationRealm\n      --infinispan-password string    The Infinispan Server password\n      --infinispan-sasl string        The Infinispan Server SASL Mechanism, example: PLAIN\n      --infinispan-url string         The Infinispan Server URI, example: infinispan-server:11222\n      --infinispan-user string        The Infinispan Server username\n      --insecure-image-registry       Indicates that the Service image points to insecure image registry\n      --kafka-instance string         The Kafka cluster external URI, example: my-kafka-cluster. When set, enables events for Jobs Service.\n      --kafka-url string              The Kafka cluster external URI, example: my-kafka-cluster:9092. When set, enables events for Jobs Service.\n      --limits strings                Resource limits for the pod. Valid values are 'cpu' and 'memory'. For example 'cpu=1'. Can be set more than once.\n  -p, --project string                The project name where the service will be deployed\n      --replicas int32                Number of pod replicas that should be deployed. (default 1)\n      --requests strings              Resource requests for the pod. Valid values are 'cpu' and 'memory'. For example 'cpu=1'. Can be set more than once.\n      --secret-env stringArray        Secret Key/Pair value environment variables that will be set to the service. For example 'MY_VAR=secretName#secretKey'. Can be set more than once.\n\nGlobal Flags:\n  -o, --output string   output format (when defined, 'json' is supported)\n  -v, --verbose         verbose output\n      --version         display version\n\n"}

{"level":"error","T":"2020-07-31T10:54:09.697Z","logger":"jenkins-cucumber-666b","msg":"Error in step 'Install Kogito Trusty with 1 replicas': exit status 1"}

{"level":"info","T":"2020-07-31T10:54:09.697Z","logger":"jenkins-cucumber-666b","msg":"Step Kogito Trusty has 1 pods running within 10 minutes"}

{"level":"error","T":"2020-07-31T10:54:09.773Z","logger":"jenkins-cucumber-666b","msg":"Error in scenario 'Install Kogito Trusty': exit status 1"}
```

Unfortunately I have some issues to run the smoke test locally,  is there anything you would suggest looking at the logs?

Reply:
@r00ta I tried to run these tests using your branch and both worked fine.